stories/topics/proc-gcp-deleting-backups-playbook.adoc

```diff
@@ -50,21 +31,23 @@ gcp_backups_delete:
     gcp_bucket_backup_name:
 ----
 
-With backup_names like ["backup1","backup2"]
+Parameter *backup_names* MUST specify an array of strings.  For example: ["backup1","backup2"].
```

Comment:
backuup_names -> do you mean gcp_backups_delete? I can't find a backup_names.

The backup-names  (in backticks) parameter must specify and array of strings, for example, ["backup1", "backup2"]

Reply:
Fixed `backup_names`